src/applications/edu-benefits/tests/0994/e2e/00-all-fields.e2e.spec.js

```diff
@@ -0,0 +1,140 @@
+import fs from 'fs';
+import path from 'path';
+
+const E2eHelpers = require('../../../../../platform/testing/e2e/helpers');
+const Timeouts = require('../../../../../platform/testing/e2e/timeouts');
+const FormsTestHelpers = require('../../../../../platform/testing/e2e/form-helpers');
+const Auth = require('../../../../../platform/testing/e2e/auth');
+const ENVIRONMENTS = require('../../../../../site/constants/environments');
+
+import {
+  completeFormPage,
+  completeAlreadySubmitted,
+  completeMilitaryService,
+  completeEducationHistory,
+  completeHighTechWorkExp,
+  getTrainingProgramsChoice,
+  completeTrainingProgramChoice,
+  completeTrainingProgramsInformation,
+  completeContactInformation,
+  completeBankInformation,
+  completeReviewAndSubmit,
+  returnToBeginning,
+} from './vet-tec-helpers';
+
+const dirName = path.join(__dirname, '../schema/');
+const startUrl = '/education/apply-for-education-benefits/application/0994/';
+
+const authentication = client => {
+  const token = Auth.getUserToken();
+
+  Auth.logIn(token, client, startUrl, 3);
+
+  // Ensure introduction page renders.
+  client.assert
+    .title('Apply for education benefits: VA.gov')
+    .waitForElementVisible('.schemaform-start-button', Timeouts.verySlow)
+    .axeCheck('.main')
+    .click('.schemaform-start-button');
+
+  E2eHelpers.overrideVetsGovApi(client);
+  FormsTestHelpers.overrideFormsScrolling(client);
+  E2eHelpers.expectNavigateAwayFrom(client, '/introduction');
+};
+
+const e2eTests = (client, formData) => {
+  // Benefits eligibility
+  // Personal Information
+  completeFormPage('/applicant/information', client, formData);
```

Comment:
probably okay to remove `formData` from this call if it's there's no form filler function passed that uses it.

Reply:
thanks, missed that